src/libraries/System.Net.Security/src/System/Net/Security/TlsFrameHelper.cs

```diff
@@ -200,6 +201,9 @@ version switch
                 SslProtocols.Tls12 => s_protocolMismatch12,
                 SslProtocols.Tls11 => s_protocolMismatch11,
                 SslProtocols.Tls => s_protocolMismatch10,
+#pragma warning disable 0618
```

Comment:
why is this warning disable needed?

Reply:
Ssl2 & Ssl3 are marked as obsolete. So we need that everywhere when used.